Heroes.ReplayParser/ParseOptions.cs

```diff
@@ -31,6 +31,14 @@ public class ParseOptions
             ShouldParseDetaileBattleLobby = true,
             ShouldParseMessageEvents = true
         };
+        
+        public static ParseOptions MediumParsing => new ParseOptions()
```

Comment:
Is there a good reason why we shouldn't just set these as the defaults (and set the others accordingly)?

Reply:
Either way works.  I just need the options set on Medium.